drivers/gpio/gpio_bd8lb600fs.c

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (c) 2022 SILA Embedded Solutions GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT rohm_bd8lb600fs
+
+/**
```

Comment:
I would suggest removing doxygen from this file.

Reply:
done